elements/pfe-clipboard/README.md

```diff
@@ -82,6 +82,7 @@ Available hooks for styling:
 | `--icon--Width` | `16px` | `icon` |
```

Comment:
```suggestion
| `--pfe-clipboard--icon--Width` | `16px` | `icon` |
```

^ These are all prefixed with pfe-clipboard

Reply:
Ahh, yes Thank you!